docs/.vuepress/public/feeds/development-atom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title>Rundeck Development Updates</title>
+  <link href="https://docs.rundeck.com/feeds/development.xml" rel="self" />
+  <link href="https://docs.rundeck.com/docs" />
+  <updated>2025-11-06T23:24:06.921Z</updated>
+  <id>https://docs.rundeck.com/feeds/development</id>
+  <subtitle>Recent merged pull requests and development updates from Rundeck</subtitle>
+  <entry>
+    <title>Multiline Job Options (Beta)</title>
+    <link href="https://docs.rundeck.com/docs/history/updates/" />
```

Comment:
should the links in the feeds go to the heading for the correct item?

Reply:
No.  Just the page.   The heading links that vuepress builds aren't consistent enough to know them ahead of time.